quickstarts/microsoft.databricks/databricks-workspace-with-managed-svc-customer-managed-keys/README.md

```diff
@@ -0,0 +1,31 @@
+---
+description: This template allows you to create an Azure Databricks workspace with managed services and CMK.
+page_type: sample
+products:
+- azure
+- azure-resource-manager
+urlFragment: databricks-workspace-with-managed-svc-customer-managed-keys
+languages:
+- bicep
```

Comment:
I think if the only file being deployed is JSON, bicep should not be listed as a language in the readme.